src/ui/public/config/defaults.js

```diff
@@ -228,6 +228,19 @@ export default function configDefaultsProvider() {
       value: 5000,
       description: 'The time in milliseconds which an information notification ' +
         'will be displayed on-screen for. Setting to Infinity will disable.'
-    }
+    },
+    'url:warnLength': {
+      value: 1900,
+      description: `
+        When the application url reaches this length we will start warning about
+        potential issues. Internet Explorer supports urls up to
+        <a href="https://support.microsoft.com/en-us/kb/208427">2,083 characters</a> long.
+        If IE compatibility is not important this can probably be disabled (set to 0).
+      `,
+    },
+    'url:limit': {
+      value: 2082,
```

Comment:
Weren't we seeing IE fail before this point? I thought 2048 was the magic number there...